runtime/runtime/src/actions.rs

```diff
@@ -1127,6 +1129,22 @@ fn check_transfer_to_nonexisting_account(
     }
 }
 
+#[cfg(feature = "test_features")]
+fn apply_recorded_storage_garbage(
+    function_call: &FunctionCallAction,
+    state_update: &mut TrieUpdate,
+) {
+    if let Some(garbage_size_mbs) = function_call
+        .method_name
+        .strip_prefix("internal_record_storage_garbage_")
+        .and_then(|suf| suf.parse::<usize>().ok())
+    {
+        if state_update.trie.record_storage_garbage(garbage_size_mbs) {
+            tracing::warn!(target: "runtime", %garbage_size_mbs, "Generated storage proof garbage");
+        }
+    }
+}
+
 #[cfg(test)]
 mod tests {
 
```

Comment:
It would be nice to document that we have a feature like this so that anyone who stumbles upon the code can figure out what they are looking at.

Reply:
Added a link to this PR, that should be enough: 39cd8dc13